src/Configuration.ts

```diff
@@ -41,3 +41,5 @@ export interface EasyLessOptions extends Less.Options {
   // outputSourceFiles?: boolean;
   // sourceMapFilename?: string;
 }
+
+export type Preprocessor = (document: vscode.TextDocument) => vscode.TextDocument;
```

Comment:
As mentioned above, I think the input and output should be `string` rather than `TextDocument`.